Makefile

```diff
@@ -9,21 +9,24 @@ push/all: push/integreatly push/amqstreams push/3scale push/rhsso push/codeready
 
 push/integreatly:
 	operator-courier verify integreatly
-	operator-courier push integreatly/ $(REPO) integreatly $(INTEGREATLY_VERSION) "$(AUTH_TOKEN)"
+	-operator-courier push integreatly/ $(REPO) integreatly $(INTEGREATLY_VERSION) "$(AUTH_TOKEN)"
 
```

Comment:
Any reason we are adding '-' to these commands? This will ignore errors right?

Reply:
yeah, so if it tries to push one that's already there, it will still move on to the next one.